### DISCUSSION TITLE
How can I add new clothes to Monika?

### DISCUSSION THREAD

**Discussion by UE1dj** [Monika-After-Story/MonikaModDev#8410] [QA - tech support + general] [answered]
Hello, it's me again, just a question, what can I do so that Monika can wear other clothes, such as the black dress, Christmas clothes and more?
But I'm not talking about the spritepacks.

**Comment by lunulae** [comment, marked as answer]
Some clothes in MAS are unlocked by being present for a certain event. The black dress unlocks if you take her on a date on her birthday, the white sundress is unlocked for Valentine's Day, the Christmas clothes are unlocked during the Christmas season, and so on.

If you have a question about a specific outfit, please let me know. :D

**Comment by UE1dj** [comment]
Oh okay, thanks.

**Comment by UE1dj** [comment]
So, how can i unlock the Christmas clothes or the unlock is automatic?

**Comment by lunulae** [comment]
I believe the unlock is automatic once she puts up the decorations. If my memory is wrong on that though, it'll be on Christmas Eve.
Note that if you have less than 1000 affection, you may not be able to select them after the Christmas season, but they will remain unlocked for when you cross that threshold. (So you won't have to wait for the holiday again)
